plexapi/livetv.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+import os
+from urllib.parse import quote_plus, urlencode
+import requests
+
+from plexapi import media, utils, settings, library
+from plexapi.base import PlexObject, Playable, PlexPartialObject
+from plexapi.exceptions import BadRequest, NotFound
+from plexapi.video import Video
+from requests.status_codes import _codes as codes
+
+
+@utils.registerPlexObject
+class IPTVChannel(Video):
+    """ Represents a single IPTVChannel."""
+
+    TAG = 'Directory'
+    TYPE = 'channel'
+    METADATA_TYPE = 'channel'
+
+    def _loadData(self, data):
+        self._data = data
+        self.guid = data.attrib.get('id')
+        self.thumb = data.attrib.get('thumb')
+        self.title = data.attrib.get('title')
+        self.type = data.attrib.get('type')
+        self.items = self.findItems(data)
+
+
+@utils.registerPlexObject
+class Recording(Video):
+    """ Represents a single Recording."""
+
+    TAG = 'MediaSubscription'
+
+    def _loadData(self, data):
+        self._data = data
+        self.key = data.attrib.key('key')
+        self.type = data.attrib.key('type')
+        self.targetLibrarySectionId = data.attrib.get('targetLibrarySectionId')
+        self.createdAt = utils.toDatetime(data.attrib.get('createdAt'))
+        self.title = data.attrib.get('title')
+        self.items = self.findItems(data)
+
+    def delete(self):
+        self._server.query(key='/media/subscription/' + self.key, method=self._server._session.delete)
+
+
+@utils.registerPlexObject
+class ScheduledRecording(Video):
+    """ Represents a single ScheduledRecording."""
+
+    TAG = 'MediaGrabOperation'
+
+    def _loadData(self, data):
+        self._data = data
+        self.mediaSubscriptionID = data.attrib.get('mediaSubscriptionID')
+        self.mediaIndex = data.attrib.get('mediaIndex')
+        self.key = data.attrib.key('key')
+        self.grabberIdentifier = data.attrib.get('grabberIdentifier')
+        self.grabberProtocol = data.attrib.get('grabberProtocol')
+        self.deviceID = data.attrib.get('deviceID')
+        self.status = data.attrib.get('status')
+        self.provider = data.attrib.get('provider')
+        self.items = self.findItems(data)
+
+
+@utils.registerPlexObject
```

Comment:
Your using the decorator so the object can be built. Have you checked that this dont conflict with other xml object that uses the same tag (so the correct class is built)

Reply:
This is the only place where the 'Setting' TAG is used